.github/workflows/tools-api-docs.yml

```diff
@@ -0,0 +1,41 @@
+name: nf-core/tools dev API docs
+# Run on push and PR to test that docs build
+on:
+  push:
+    branches:
+      - dev
+    paths:
+      - nf_core/**/*.py
+  release:
+    types: [published]
+  workflow_dispatch:
+    inputs:
+      ref_name:
+        description: "The branch or tag to build the API docs for"
+        required: true
+        default: "dev"
+
+# Cancel if a newer run is started
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  api-docs:
+    name: trigger API docs build on website repo
+    runs-on: ubuntu-latest
+    steps:
+      - name: trigger API docs build
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{ secrets.nf_core_bot_auth_token }}
+          script: |
+            await github.rest.actions.createWorkflowDispatch({
+              owner: 'nf-core',
+              repo: 'website',
+              workflow_id: 'add-tools-api-docs.yml',
```

Comment:
Where is this workflow located? I can't find it

Reply:
It's in this PR: https://github.com/nf-core/website/pull/2268/files#diff-dfae69a640999485f051208d7f3555435d4302d04a60f6224818a7aa48f98e10